src/main/java/net/dries007/tfc/common/entities/aquatic/Penguin.java

```diff
@@ -21,9 +27,12 @@ public class Penguin extends AmphibiousAnimal
     public final AnimationState walkingAnimation = new AnimationState();
     public final AnimationState swimmingAnimation = new AnimationState();
 
+    private final Supplier<? extends SoundEvent> ambient;
```

Comment:
Why is this on a field and not just direct?

Reply:
Well, I apparently didn't notice that the WildAnimal class already had a field for this. I'll just remove that